### ISSUE TITLE
Suggestion: Don't show progress bar when pausing/unpausing

### ISSUE THREAD

**Issue by H2SO4JB** [yuliskov/SmartYouTubeTV#370]
Hi,

I have a suggestion for a tweak. I've always wanted to be able to pause a video, and then unpause it, without the progress bar to pop-up.

It's often I need to pause a video to see something fast, and then it's just annoying to have to press the up button every time to hide the progress bar or wait for two seconds for it to hide automatically, to see the whole picture. Or even worse in the official YouTube app, you must press the up button twice.

Sure, the progress bar is hidden faster in SYTV, but I don't want to see it at all when I just pause a video. I mean, if I want to see the progress bar, I can just press the down button at any time.

Does anyone agree with me? If so, is it possible to add a tweak to SYTV that does this?
And call it something like this: "Don't show progress bar when pausing/unpausing"

Thanks in advance...

**Comment by versus2** [comment]
I totally agree with you.  And it was only recently introduced.

**Comment by H2SO4JB** [comment]
What do you mean? Is this feature available now?

**Comment by versus2** [comment]
No.i It was available until last update. I also complained to the author in a separate thread . My grudge was about  the  update breaking what was good  before  And speed selection also. Was perfect for analyzing action sports.Sent from my Huawei phone-------- Original Message --------Subject: Re: [yuliskov/SmartYouTubeTV] Suggestion: Don't show progress bar when pausing/unpausing (#370)From: H2SO4JB To: yuliskov/SmartYouTubeTV CC: versus2 ,Comment What do you mean? Is this feature available now?
